app/src/main/res/values/strings_settings.xml

```diff
@@ -66,8 +66,8 @@
     <string name="windowbar_mode_nav">Hide navigation bar</string>
     <string name="windowbar_mode_both">Hide both</string>
 
-    <string name="gestures_left">Swipe left</string>
-    <string name="gestures_right">Swipe right</string>
+    <string name="gestures_left">Swipe &lt;---</string>
+    <string name="gestures_right">Swipe ---&gt;</string>
```

Comment:
I'm still troubled by the string here. Would this make it more understandable? We may be using a better indication here, however — and that may be enough.

Reply:
ok i am not sure how to talk back commit or unchanged can you discard this file .